.github/actions/publish/action.yml

```diff
@@ -29,17 +30,42 @@ runs:
       with:
         platforms: linux/amd64,linux/arm64/v8,linux/arm/v7,linux/386
     - name: Login to Docker
+      shell: bash
+      run: echo $DOCKER_TOKEN | docker login --username $DOCKER_USERNAME --password-stdin
+
+    - name: Set up goreleaser
       shell: bash
       run: |
-        echo $DOCKER_TOKEN | docker login --username $DOCKER_USERNAME --password-stdin
+        CONTAINER_ID="$(
```

Comment:
Comment explaining why the normal goreleaser-action can't be used would be great